src/neo/Trie/MPT/MPTNode/FullNode.cs

```diff
@@ -0,0 +1,62 @@
+using Neo.Cryptography;
+using Neo.IO;
+using System.IO;
+
+namespace Neo.Trie.MPT
+{
+    public class FullNode : MPTNode
+    {
+        public const int CHILD_COUNT = 17;
+
+        public MPTNode[] Children = new MPTNode[CHILD_COUNT];
+
+        public override int Size
+        {
+            get
+            {
+                var size = 1;
+                for (int i = 0; i < Children.Length; i++)
+                {
+                    size += Children[i].GetHash().Length;
+                }
```

Comment:
As you use `writer.WriteVarBytes(Children[i].GetHash())` in `Serialize`, should change the size?
```csharp
size = base.Size + Children[i].GetHash().GetVarSize() * CHILD_COUNT
```
or 
```csharp
size = base.Size
for(var child in Children)
   size += child.Size
```

Reply:
removed